components/Cards/examples/CardsShapedEditReorderExample.swift

```diff
@@ -33,10 +33,14 @@ class ShapedCardCollectionCell: MDCCardCollectionCell {
     let shapeGenerator = MDCRectangleShapeGenerator()
     shapeGenerator.topLeftCorner = MDCCutCornerTreatment(cut: 20)
     self.shapeGenerator = shapeGenerator
+
+    self.isAccessibilityElement = true
+    self.accessibilityTraits = .button
+    self.accessibilityLabel = "Shaped collection cell"
```

Comment:
Nit: `self.` is not necessary on lines 37-39

Reply:
From the Swift style guide: https://google.github.io/swift/#initializers

> For clarity, initializer arguments that correspond directly to a stored property have the same name as the property. Explicit self. is used during assignment to disambiguate them.

Reply:
I think this guidance is a bit fuzzy though.

Reply:
I think to be self-consistent it's likely wise for us to always use `self.` in initializers because otherwise it could be a mish-mash of both.

Reply:
Ah, I didn't realize we had a Swift style guide :) Makes sense!